clients/java/src/main/java/io/camunda/zeebe/client/impl/ZeebeClientBuilderImpl.java

```diff
@@ -441,49 +447,49 @@ public ZeebeClientBuilder keepAlive(final Duration keepAlive) {
   }
 
   @Override
-  public ZeebeClientBuilder withInterceptors(final ClientInterceptor... interceptors) {
+  public CamundaClientBuilder withInterceptors(final ClientInterceptor... interceptors) {
     this.interceptors.addAll(Arrays.asList(interceptors));
     return this;
   }
 
   @Override
-  public ZeebeClientBuilder withJsonMapper(final JsonMapper jsonMapper) {
+  public CamundaClientBuilder withJsonMapper(final JsonMapper jsonMapper) {
     this.jsonMapper = jsonMapper;
     return this;
   }
 
   @Override
-  public ZeebeClientBuilder overrideAuthority(final String authority) {
+  public CamundaClientBuilder overrideAuthority(final String authority) {
     overrideAuthority = authority;
     return this;
   }
 
   @Override
-  public ZeebeClientBuilder maxMessageSize(final int maxMessageSize) {
+  public CamundaClientBuilder maxMessageSize(final int maxMessageSize) {
     this.maxMessageSize = maxMessageSize;
     return this;
   }
 
   @Override
-  public ZeebeClientBuilder defaultJobWorkerStreamEnabled(final boolean streamEnabled) {
+  public CamundaClientBuilder defaultJobWorkerStreamEnabled(final boolean streamEnabled) {
     this.streamEnabled = streamEnabled;
     return this;
   }
 
   @Override
-  public ZeebeClientBuilder useDefaultRetryPolicy(final boolean useDefaultRetryPolicy) {
+  public CamundaClientBuilder useDefaultRetryPolicy(final boolean useDefaultRetryPolicy) {
     this.useDefaultRetryPolicy = useDefaultRetryPolicy;
     return this;
   }
 
   @Override
-  public ZeebeClientBuilder preferRestOverGrpc(final boolean preferRestOverGrpc) {
+  public CamundaClientBuilder preferRestOverGrpc(final boolean preferRestOverGrpc) {
     this.preferRestOverGrpc = preferRestOverGrpc;
     return this;
   }
 
   @Override
-  public ZeebeClient build() {
+  public CamundaClient build() {
```

Comment:
❓ Just wondering: is this less of a breaking change for users than returning a `CamundaFuture` instead of a `ZeebeFuture`?

Reply:
You are right, I think we need to take the same approach used for the command here. Creating a new method with a different return type

Reply:
We could also let the `Zeebe...` classes simply still return their `Zeebe...` counterparts wherever possible to be as minimally invasive as possible. Then we should only have very few points where it "collides" (e.g. `FinalCommandStep`).

Reply:
Or, just duplicate the code here

Reply:
Something like:

```
public final class CamundaClientBuilderImpl implements
    CamundaClientBuilder, CamundaClientConfiguration 
```

and 

```
@Deprecated
public class ZeebeClientBuilderImpl implements ZeebeClientBuilder, ZeebeClientConfiguration 
```

Reply:
Yeah, I was also just thinking:
Can we simply duplicate the code completely into the `io.camunda.client` package, rename everything there and at the critical conversion points take care of having both and handle it accordingly. With 8.8, we simply remove the `io.camunda.zeebe.client` package, rename the conversion point classes like `FinalCommandStep`, and remove their deprecated methods like `send()`.

Reply:
It's a bit ugly for now and for patching later though, as the code is duplicated completely.

Reply:
Let's leave this as an open point for Monday's team meeting. Deeper we go, deeper I'm thinking about just duplicating everythin 🤣